wmi-adapter/wmi.resource.ps1

```diff
@@ -9,10 +9,28 @@ param(
     $stdinput
 )
 
+trap {
```

Comment:
Would be good to have a comment about what this `trap` does.

Reply:
ok, will add comment